src/System.Binary.Base64/System/Binary/Base64Decoder.cs

```diff
@@ -1,11 +1,12 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Buffers;
 using System.Runtime.CompilerServices;
 
 namespace System.Binary.Base64
 {
-    public static class Base64Encoder
+    public class Base64Decoder : ITransformation
```

Comment:
this needs to be a singleton. Make the default ctor private, create a static singleton instance, and expose the instance as a static property. See https://github.com/dotnet/corefxlab/blob/master/src/System.Buffers.Primitives/System/Buffers/BufferPool.cs#L8